.buildkite/pipeline.yaml

```diff
@@ -26,7 +26,7 @@ steps:
 
   - label: ":mac: test-static-sanitized.sh (master only)"
     command: .buildkite/test-static-sanitized.sh
-    branches: "master"
+    #branches: "master"
```

Comment:
?

Reply:
I'm planning on reverting this after a build passes on OSX